section3/reflection.md

```diff
@@ -2,16 +2,32 @@
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
 
+* I tried out the Pomodoro technique for Section 3, but found that I either estimated too much time for a segment (EX: part A, exercise 1 etc), or not enough time (Ex: hashes.rb). I think that the act of _trying_ to estimate time for a section of work is a skill I will pick up as I go.  Also, even if I didn't stick to exactly 25 minutes, I found that taking the break was really helpful in keeping me engaged.
+
 2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
 
+* I applied the Pomodoro technique to section 3 of the project, by skimming the tasks and creating a pom for each segment.  I found that the timer really did help me stay focused, but if I finished a segment early, or took longer I wasn't sure what to do with the rest of my time.  Should I start the next Pom if I finished early? Should I create another Pom if I didn't finish a segment in the allotted 25 minutes?
```

Comment:
Yeah I think that's a great reflection and question - maybe it depends on what is next on the list? Is the next task relatively small/do you have the energy to go straight into it? Or is it a really big one, so maybe you take a break to reenergize before going  into it.